third_party/nvfuser/csrc/python_frontend/python_bindings.cpp

```diff
@@ -19,37 +19,33 @@
 
 namespace nvfuser::python_frontend {
 
-std::vector<bool> computeContiguity(
+std::vector<c10::optional<bool>> computeContiguity(
     const std::vector<int64_t>& sizes,
     const std::vector<int64_t>& strides) {
   TORCH_CHECK(
       sizes.size() == strides.size(),
       "compute_contiguity: Sizes and strides must have the same number of dimensions");
   auto not_broadcast = [&](auto i) { return strides[i] != 0 && sizes[i] != 1; };
-  auto irange = c10::irange(sizes.size());
-  auto no_b_size = std::count_if(irange.begin(), irange.end(), not_broadcast);
-  std::vector<bool> contiguity(no_b_size);
+  std::vector<c10::optional<bool>> contiguity(sizes.size(), c10::nullopt);
   if (contiguity.size() == 0) {
     return contiguity;
   }
   int64_t last = sizes.size() - 1;
   for (; last >= 0; --last) {
     if (not_broadcast(last)) {
+      contiguity[last] = (strides.at(last) == 1);
       break;
     }
   }
-  contiguity[no_b_size - 1] = (strides.at(last) == 1);
-  int64_t no_broadcast_i = 0;
   for (int64_t i = 0; i < last;) {
     if (not_broadcast(i)) {
       auto l = i++;
       for (; i <= last; i++) {
         if (not_broadcast(i)) {
           break;
-        }
+	}
```

Comment:
nit: Please revert this white space change (and consider using a modern editor rather than vim😈😈😈😈)

Reply:
Thanks for catching this~~ somehow my clangformat changes didn't get pushed.

> and consider using a modern editor rather than vim

I'm going to unfriend you on all my social media.